pkg/conversion/convert.go

```diff
@@ -587,6 +591,9 @@ func convertWindupPerformToAnalyzer(perform windup.Iteration, where map[string]s
 			ret["effort"] = i
 		}
 
+		if len(hint.Tag) != 0 {
```

Comment:
isnt `hint.Tag` really an application tag?

Reply:
What else would it be?

Reply:
It is a particular tag, for a particular violation/incident or a rule, I can only assume that we want to map this to a rule label, right ?